vitrina/datasets/templates/vitrina/datasets/list.html

```diff
@@ -1,12 +1,423 @@
+                           class="tag {% if filterType == "organization" %}is-success{% endif %}{% if filterType == "category" %}is-info{% endif %}">{{ fltr.title }}
+                            <i class="fas fa-times"></i>
+                        </a>
+                    {% endfor %}
+                {% endfor %}
+                <br/>
+                <br/>
+            {% endif %}
+
+            {#{% include '../component/pagination.twig' %}#}
+
+            {% for i in page_obj %}
+                <article class="media">
+                    {#<div class="media-content single-item">#}
+                    <div class="media-content">
+                        <div class="content">
+                            <div>
+                                <strong><a href="{{ i.get_absolute_url }}"
+                                           class="dataset-list-title">{% if locale == "lt" %}{{ i.title }}{% else %}{% if i.title_en != "" %}{{ i.title_en }}{% else %}{{ i.title }}{% endif %}{% endif %}</a></strong>
+                                <div class="is-pulled-right">
+                                    <a target="_blank"
+                                       href="https://www.facebook.com/sharer/sharer.php?u=https://data.gov.lt/{{ radix }}/{{ i.slug }}">
+                                    <span class="icon">
+                                        <i class="fab fa-facebook-square"></i>
+                                    </span>
+                                    </a>
+                                    {#<a target="_blank" href="https://www.linkedin.com/shareArticle?mini=true&url=https://data.gov.lt/{{ radix }}/{{ i.slug }}&title={{ i.title }}">#}
+                                    {#<span class="icon">#}
+                                    {#<i class="fab fa-linkedin"></i>#}
+                                    {#</span>#}
+                                    {#</a>#}
+                                    {#<a target="_blank" href="http://twitter.com/share?text={{ i.title }}&url=https://data.gov.lt/{{ radix }}/{{ i.slug }}">#}
+                                    {#<span class="icon">#}
+                                    {#<i class="fab fa-twitter-square"></i>#}
+                                    {#</span>#}
+                                    {#</a>#}
+                                </div>
+                            </div>
+                            {% if i.organization_title %}
+                                <p>
+                                    <small class="dataset-list-organization">{{ i.organization_title }}</small>
+                                </p>
+                            {% endif %}
+                            <p class="dataset-list-description">
+                                {% if locale == "lt" %}{{ i.description }}{% else %}{% if i.description_en != "" %}{{ i.description_en }}{% else %}{{ i.description }}{% endif %}{% endif %}
```

Comment:
Remove translatable model logic from templates, this will come with https://github.com/atviriduomenys/katalogas/issues/61.